src/ansys/fluent/core/launcher/watchdog.py

```diff
@@ -158,7 +158,7 @@ def launch(main_pid: int, sv_port: int, sv_password: str, sv_ip: Optional[str] =
         # Configure logger for Watchdog process
         log_config = pyfluent.logging.get_default_config()
         log_config["handlers"]["pyfluent_file"][
-            "filename"
+            "file_name"
```

Comment:
Noticed an issue when testing locally:
```suggestion
            "filename"
```
This one needs to match the one we use in the `logging` module, either change both, or don't change this one

Reply:
Done. Thank you.